app/server/middleware/attachmentDeleteRouter.ts

```diff
@@ -0,0 +1,64 @@
+import { Router } from "express";
+import { performQuery } from "./graphql";
+import { Storage } from "@google-cloud/storage";
+import config from "../../config";
+
+export const attachmentDeleteRouter = Router();
+
+const attachmentDetailsQuery = `query AttachmentDetailsQuery($attachmentId: ID!){
+  attachment(id: $attachmentId) {
+    file
+  }
+}`;
+
+const discardProjectAttachmentFormChangeMutation = `mutation discardProjectAttachmentFormChangeMutation(
+  $input: DiscardProjectAttachmentFormChangeInput!
+  $connections: [ID!]!
+) {
+  discardProjectAttachmentFormChange(input: $input) {
+    formChanges {
+      projectRevisionByProjectRevisionId {
+      }
+      id @deleteEdge(connections: $connections)
+    }
+  }
+}`;
+
+export const handleDelete = async (req, res, next) => {
+  try {
+    //  get info about the attachment
+    const attachmentQueryVariables = {
+      attachmentId: req.params.attachmentId,
+    };
+    const result = await performQuery(
+      attachmentDetailsQuery,
+      attachmentQueryVariables,
+      req
+    );
+
+    const {
+      data: {
+        attachment: { file },
+      },
+    } = result;
+
+    // do the delete mutation
+    console.log("$$$$$$req.body.variables", req.body.variables);
+    const brianna = await performQuery(
+      discardProjectAttachmentFormChangeMutation,
+      req.body.variables,
+      req
+    );
+    console.log("$$$$$$$$$$$brianna", brianna);
+    const storageClient = new Storage();
+    const bucketName = config.get("attachmentsBucket");
+
+    const response = await storageClient.bucket(bucketName).file(file).delete();
+
+    return res.sendStatus(response[0].statusCode);
+  } catch (error) {
+    next(error);
+  }
+};
+
+attachmentDeleteRouter.get("/delete/:attachmentId", handleDelete);
```

Comment:
Your router is expecting a `GET` request, but the request uses a `DELETE` method in `AttachmentTableRow.tsx`

```suggestion
attachmentDeleteRouter.delete("/delete/:attachmentId", handleDelete);
```